install-daemon.sh

```diff
@@ -155,6 +161,50 @@ function check_os_comp {
     print_error "Unsupported OS"
     exit 1
   fi
+  if [ "$OS" == "debian" ] || [ "$OS" == "ubuntu" ] || [ "$OS" == "zorin" ]; then
+    apt-get update
+
+    # install virt-what
+    apt-get install -y virt-what
+	
+   elif [ "$OS" == "centos" ]; then
+	
+    if [ "$OS_VER_MAJOR" == "7" ]; then
+      yum install -y update
```

Comment:
Hmm, is this supposed to be `yum-y update`?

Reply:
Yes correct. Another typo.😊